ironfish/src/networks/network.ts

```diff
@@ -15,11 +15,11 @@ export class Network {
 
   private miningRewardCachedByYear = new Map<number, number>()
 
-  constructor(definition: NetworkDefinition, consensus: Consensus) {
+  constructor(definition: NetworkDefinition) {
     this.id = definition.id
     this.default = isDefaultNetworkId(definition.id)
     this.definition = definition
```

Comment:
Doesn't seem like this is an issue rn but we do create a new object from consensus params in the `Consensus` constructor. Wondering if it would make sense to do the same here?

Reply:
Fixed